utils/github/backport.py

```diff
@@ -20,9 +20,26 @@ def __init__(self, token, owner, name, team):
     def getPullRequests(self, from_commit):
         return self._gh.get_pull_requests(from_commit)
 
-    def execute(self, repo, until_commit, number, run_cherrypick):
+    def getBranchesWithLTS(self, release_branches):
+        # it's convenient to return branches keeping their precedence order
+        branches = []  # [(branch_name, base_commit)]
+        for pull_request in self._gh.find_pull_requests("release-lts"):
+            if not pull_request['merged'] and not pull_request['closed']:
+                for branch in release_branches:
+                    if branch[0] == pull_request['headRefName']:
+                        branches.append(branch)
+                        break
+        return branches
+
+    def execute(self, repo, until_commit, number, run_cherrypick, find_lts=False):
         repo = LocalRepo(repo, 'origin', self.default_branch_name)
+        all_branches = repo.get_release_branches()
+
+        # assume that LTS branches are always older than recent ones
```

Comment:
We define the version as LTS as soon as it is released, not later.
The of LTS versions can intersect the set of stable versions.